airflow/config_templates/default_airflow.cfg

```diff
@@ -61,9 +62,15 @@ logging_level = INFO
 logging_config_class =
 
 # Log format
+# we need to escape the curly braces by adding an additional curly brace
 log_format = [%%(asctime)s] {{%%(filename)s:%%(lineno)d}} %%(levelname)s - %%(message)s
 simple_log_format = %%(asctime)s %%(levelname)s - %%(message)s
 
+# Log filename format
+# we need to escape the curly braces by adding an additional curly brace
+log_filename_template = {{{{ ti.dag_id }}}}/{{{{ ti.task_id }}}}/{{{{ ts }}}}/{{{{ try_number }}}}.log
+log_processor_filename_template = {{{{ filename }}}}.log
```

Comment:
@NielsZeilemaker I think these additional curly braces on LL71-72 (changing from double curly braces to 4 curly braces) are incorrect. In my environment, the break Jinja2 parsing. The additional curly braces on L540 (changing from single curly braces to double curly braces) are correct. I suspect this was just a mistake when making the changes, like maybe a wayward sed command. 😄

Reply:
@Fokko I think there already are tests that covers this:

- https://github.com/apache/incubator-airflow/blob/master/tests/utils/test_log_handlers.py#L181
- https://github.com/apache/incubator-airflow/blob/master/tests/utils/log/test_file_processor_handler.py#L35

The problem is that the config_templates don't match the tests.

Reply:
@Fokko @NielsZeilemaker I finally figured out what is going on here, and it's not a bug. 😬 Those strings with additional curly braces end up getting parsed as template strings, formatted, and written out in `airflow.cfg`. So the source default config has to contain something like `{{{{ filename }}}}.log` in order to get the corresponding entry in `airflow.cfg` to be `{{ filename }}.log`. 

Sorry for the false alarm.